master/reference/public-cloud/azure.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Deploying Calico on Azure
+no_canonical: true
+---
+
+{{site.prodname}} in [Microsoft Azure][Azure]{:target="_blank"} supported in [policy-only][PolicyMode] mode.
+Networking part covered by Azure user-defined routes. {{site.prodname}} IPAM needs
+to be configured in host-local mode and used in conjunction with Kubernetes pod CIDR assignments.
+
+#### Routing Traffic
+
+Only available option for traffic routing in Azure with {{site.prodname}} is [Azure user-defined routes][AzureUDR]{:target="_blank"}.
+
+##### Azure user-defined routes
+
+To use Azure routing you must create [Azure route table][AzureUDRCreate]{:target="_blank"} and associatе it with VMs subnet.
+
+
+#### Enabling IP forwarding
+
+To allow pods traffic make sure VM network interfaces have [ip forwarding enabled][AzureIPForward]{:target="_blank"} in Azure.
+
+#### Enabling Kubernetes pod CIDR assignment
+
+To enable automatic pod CIDR assignement make sure Kubernetes controller manager has allocate-node-cidrs set to true
+and proper subnet in cluster-cidr parameter. Make sure that selected pods subnet is a part of your Azure virtual network ip range.
+You also must have Kubernetes Azure cloud provider configured with your routing table name in configuration file.
+
+#### Why {{site.prodname}} networking is not supported in Azure?
```

Comment:
"Why {{site.prodname}} networking is not supported in Azure?" --> "Why doesn't Azure support {{site.prodname}} networking?"